server/osx/server_osx.py

```diff
@@ -131,10 +131,10 @@
 
 _MODIFIER_KEY_DIRECT = {
     # modifiers
-    'command': 'command',
+    'win': 'command',
     'shift': 'shift',
-    'option': 'option',
-    'control': 'control',
+    'alt': 'option',
+    'ctrl': 'control',
```

Comment:
@seananderson Could you enlighten us on what this does (you did this on commit with message ` Fix modifier keys for OS X`)

Reply:
Actually, I'm not sure. I made that change a long time ago when I was first starting out and might have just matched what I did above for `_MOD_TRANSLATION`. It doesn't seem to matter what I change the keys or values to here in terms of what happens when I run something like `Key('c-a')`. Is it possible these only affect the functionality to press and hold down or release modifier keys? I don't use that functionality. If I'm reading the code right, `_MODIFIER_KEY_DIRECT` only gets used to check whether a key is a modifier and can therefore be held down or released.

Reply:
I can confirm you are correct. I will edit the commit message to make this explicit